examples/1App_ALAEF_LCGTEP/README.md

```diff
@@ -0,0 +1,22 @@
+# 1App_ALEAF_LCGTEP
+
+## Purpose
+
+This example provides a demonstration for using WATTS to run and analyze simulations with A-LEAF (Argonne Low-carbon Electricity Analysis Framework) under various supply chain disruption scenarios. The example explores different fuel price settings to understand the impact on US nuclear capacity expansion result.
+
+
+## Code(s)
+
+- A-LEAF
+- Julia (A-LEAF dependency)
```

Comment:
Generally we haven't listed transitive dependencies like this in the list of codes involved